packages/shared/src/contexts/payment/StoreKit.tsx

```diff
@@ -118,6 +150,58 @@ export const StoreKitSubProvider = ({
     [user?.subscriptionFlags?.appAccountToken],
   );
 
+  useEffect(() => {
```

Comment:
Where does this come from, just missed events?

Reply:
Yeah, just missed events